src/aws_sagemaker/aws_sagemaker.py

```diff
@@ -221,6 +201,53 @@ async def get_object_point_clouds(self,
                                       **kwargs) -> List[PointCloudObject]:
         raise NotImplementedError
     
+    async def get_properties(
+            self,
+            *,
+            extra: Optional[Mapping[str, Any]] = None,
+            timeout: Optional[float] = None) -> Vision.Properties:
+                return Vision.Properties(
+                classifications_supported=True,
+                detections_supported=True,
+                object_point_clouds_supported=False,
+        )
+    
+
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+    
+        result = CaptureAllResult()
+        if camera_name not in self.source_cams:
+            raise Exception(
+                "Camera name given to method",camera_name, " is not one of the configured source_cams ", self.source_cams)
+        cam = self.cameras[camera_name]
+        img = await cam.get_image(mime_type=CameraMimeType.JPEG)
+        if return_image:
+            result.image = img
+        if return_classifications:
+            classifs = await self.get_classifications(img, 5)
+            result.classifications = classifs
+        if return_detections:
+            dets = await self.get_detections(img)
+            result.detections = dets 
```

Comment:
If the AWS model doesn't return classification or detections, then this will throw an exception. You'll want to check first from the properties if it even supports classifications or detections before try to call the "get_" method